### ISSUE TITLE
2.6.2 Releases missing

### ISSUE THREAD

**Issue by btreut** [stevengj/NLoptBuilder#2]
being redirected from [ReadTheDocs](https://nlopt.readthedocs.io/en/latest/NLopt_on_Windows/) to a [not found page](https://github.com/stevengj/NLoptBuilder/releases/download/v2.6.2/NLopt.v2.6.2.i686-w64-mingw32.tar.gz) for release 2.6.2, I arrived at the [main releases page](https://github.com/stevengj/NLoptBuilder/releases), where 2.6.1 is the most recent. It seems that the automated build process failed.

**Comment by btreut** [comment]
see also: [issue in NLoptBuilder](https://github.com/stevengj/NLoptBuilder/issues/2)

**Comment by christophfroehlich** [comment]
are there any news on this issue? The same happened to me..

**Comment by btreut** [comment]
I don't think so, see also issue [Windows precompiled dlls #359](https://github.com/stevengj/nlopt/issues/359)
